portfolio/src/main/webapp/style.css

```diff
@@ -1,14 +1,24 @@
-header {
-    text-align: center;
-}
-
-#home-body {
+.content-body {
     display: block;
     margin-left: auto;
     margin-right: auto;
     width: 50%;
 }
 
+footer {
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  background-color: #bebebe;
+  color: white;
+  text-align: center;
+}
+
+header {
+    text-align: center;
+}
+
 img {
     height: 500px;
```

Comment:
just occurred to me that you probably wanna say 

```
img {
  height: 500px;
  width: auto;
}
```

to maintain your image aspect ratios.

Reply:
I see, thanks!